content/blog/2016/2016-12-19-declarative-pipeline-beta.adoc

```diff
@@ -48,7 +48,7 @@ of the pipeline. Granular flow-control is extremely powerful and Scripted
 Pipeline syntax will always be part of Pipeline but it's not for everyone.
 
 Declarative Pipeline enables all users to connect simple, declarative blocks
-that define build agents (including Docker), post build actions, environment
+that define agents (including Docker), post build actions, environment
```

Comment:
if we want to remove "build" let's remove it altogether. Let's change "post build"  on same line to something like "post Pipeline" or "post run".